### ISSUE TITLE
Issue Installing Alien::XGBoost Library using Strawberry Perl

### ISSUE THREAD

**Issue by ghost** [pablrod/p5-Alien-XGBoost#5]
I'm unable to install the Alien::XGBoost library using the default installation of strawberry perl on windows 10.  I'm using the 64-bit version of strawberry perl.  When installing Alien::XGBoost, I receive an error on line 49 in the commandsequence.pm file as shown below.
```
gmake[1]: Leaving directory 'C:/STRAWB~1/cpan/build/Alien-XGBoost-0.04-1/_alien/build_wYwS'
Alien::Build::CommandSequence> + md "C:/STRAWB~1/cpan/build/Alien-XGBoost-0.04-1/blib/lib/auto/share/dist/Alien-XGBoost/dynamic"
Alien::Build::CommandSequence> + md "C:/STRAWB~1/cpan/build/Alien-XGBoost-0.04-1/blib/lib/auto/share/dist/Alien-XGBoost/bin"
Alien::Build::CommandSequence> + copy xgboost.exe C:/STRAWB~1/cpan/build/Alien-XGBoost-0.04-1/blib/lib/auto/share/dist/Alien-XGBoost/bin/
The syntax of the command is incorrect.
external command failed at C:/Strawberry/perl/site/lib/Alien/Build/CommandSequence.pm line 49.
gmake: *** [Makefile:2881: _alien/mm/build] Error 2
  PABLROD/Alien-XGBoost-0.04.tar.gz
  C:\STRAWB~1\c\bin\gmake.exe -- NOT OK
Stopping: 'install' failed for 'Alien::XGBoost'.
```

**Comment by pablrod** [comment]
Thanks for reporting this!! 

It's a bug! I forgot to quote the path. 
As you can see both `md` commands have the path double quoted (""), but not the copy command. 
I'm working to fix it and also I'm going to get travis and appveyor integrations to avoid this in the future.

This weekend this will be fixed. Meanwhile please install the previous version:
```cmd
cpanm PABLROD/Alien-XGBoost-0.03.tar.gz
```

**Comment by ghost** [comment]
Excellent.  The previous version .0.03. worked.  Thank you very much

 

Jelani

**Comment by pablrod** [comment]
Released version 0.05! and added badges!
